tempodb/backend/azure/azure_helpers.go

```diff
@@ -17,8 +18,8 @@ const (
 	uptoHedgedRequests = 2
 )
 
-func GetContainerURL(ctx context.Context, conf *Config, hedge bool) (blob.ContainerURL, error) {
-	c, err := blob.NewSharedKeyCredential(conf.StorageAccountName.String(), conf.StorageAccountKey.String())
+func GetContainerURL(ctx context.Context, cfg *Config, hedge bool) (blob.ContainerURL, error) {
```

Comment:
Rename `conf` to `cfg` to keep the naming consistent with the GCS implementation.